MQTicketService/README.md

```diff
@@ -25,14 +23,10 @@ Steps for running the application on your local machine - <b>Do not follows thes
 1. Clone this repository.
     * ```git clone "https://github.com/ibm-messaging/mq-dev-badge-sample.git"```
 1. Change directory into the TicketGenerator directory.
-    * ```cd mq-dev-badge-sample/DockerFiles/TicketGenerator```
-1. Create a new directory inside.
-    * ```mkdir jarFiles```
-1. Download all the ```.jar``` files above ^^^.
-1. Once the files have downloaded, throw them into the ```jarFiles``` directory.
-1. From the ```TicketGenerator``` folder, compile the source code with relevant class paths.
-    * Linux & Mac ```javac -cp ./jarFiles/*: com/ibm/mq/badge/*.java```
-    * Windows ```javac -cp .\jarFiles\*; com\ibm\mq\badge\*.java```
+    * ```cd mq-dev-badge-sample/MQTicketService/TicketGenerator```
+1. Build the `.jar` file required for by running the following command:
```

Comment:
Change

Build the .jar file required for by running the following command:
to
Build the application by running the following command: